middlewares/selectAndAuthzInstanceQuestion.sql

```diff
@@ -65,8 +65,8 @@ FROM
     JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
     JOIN pl_courses AS c ON (c.id = ci.course_id)
     JOIN assessment_sets AS aset ON (aset.id = a.assessment_set_id)
-    LEFT JOIN group_users AS gu ON (gu.group_id = ai.group_id)
-    LEFT JOIN groups AS gr ON (gr.id = gu.group_id AND gr.deleted_at IS NULL)
+    LEFT JOIN groups AS gr ON (gr.id = ai.group_id AND gr.deleted_at IS NULL)
+    LEFT JOIN group_users AS gu ON (gu.group_id = gr.id)
```

Comment:
While we are making this change, can we switch to `g` as the abbreviation for `groups`, to match https://github.com/PrairieLearn/PrairieLearn/blob/5dfb7b3ab2f825f76dfb05aed84a36c3c5598e5c/middlewares/selectAndAuthzAssessmentInstance.sql#L36

Reply:
Sure, I am also making all the gr -> g changes in #3157